lightningd/channel_control.c

```diff
@@ -261,6 +254,35 @@ static void handle_error_channel(struct channel *channel,
 	tal_free(forgets);
 }
 
+static void handle_error_channel(struct channel *channel,
+				 const u8 *msg)
+{
+	if (!fromwire_channel_send_error_reply(msg)) {
+		channel_internal_error(channel, "bad send_error_reply: %s",
+				       tal_hex(tmpctx, msg));
+		return;
+	}
+
+	forget(channel);
+}
+
+void forget_channel(struct channel *channel, char *why)
```

Comment:
This could be `const char *why`, saving the const-ness change in the last patch.

Reply:
updated, thanks